frontend/src/app/main/errors.cljs

```diff
@@ -96,16 +95,23 @@
                   (print-trace! error)
                   (print-data! error))))
 
-;; We receive a explicit authentication error; this explicitly clears
+;; We receive a explicit authentication error;
+;; If the uri is for workspace, dashboard or view assign the
+;; exception for the 'Oops' page. Otherwise this explicitly clears
 ;; all profile data and redirect the user to the login page. This is
 ;; here and not in app.main.errors because of circular dependency.
 (defmethod ptk/handle-error :authentication
-  [_]
-  (let [msg (tr "errors.auth.unable-to-login")
-        uri (. (. js/document -location) -href)]
-    (st/emit! (du/logout {:capture-redirect true}))
-    (ts/schedule 500 #(st/emit! (msg/warn msg)))
-    (ts/schedule 1000 #(swap! storage assoc :redirect-url uri))))
+  [e]
+  (let [msg        (tr "errors.auth.unable-to-login")
+        uri        (. (. js/document -location) -href)
```

Comment:
don't use js/document, prefer use app.util.global ns for that purpose

Reply:
fixed